src/wmc.rs

```diff
@@ -0,0 +1,566 @@
+// New Java switches are not correctly recognised by RCA, which returns some error nodes.
```

Comment:
nit: add a TODO.

Reply:
I turned the comment into a FIXME and better explained that this is an issue already fixed upstream, in the tree-sitter-java GitHub repository

Reply:
Related to the dump_root comment above:
I introduced an enum to distinguish between dumping a FuncSpace or a FileClasses struct